tests/in/bitcast.wgsl

```diff
@@ -0,0 +1,8 @@
+[[stage(vertex)]]
+fn main(
+  [[location(0)]] pos : vec4<i32>) -> [[builtin(position)]] vec4<f32> {
+  let example1: f32 = bitcast<f32>(pos.x);
```

Comment:
let's add it to `operators.wgsl`, which already has `fn builtins()`, instead of adding a new test